<a href="https://colab.research.google.com/github/kzukeri/portfolio/blob/main/Kweli-Final-Project.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

#Research Questions
1. Which food products have the highest total greenhouse gas emissions per kilogram of product, and what stages contribute most to those emissions?
2. How significant is the processing and transport stage in the total emissions of different crops? Are there food products where the emissions of post-farm stages outweigh farm-stage level emissions? [Create a meta-variable for farm-state and post-farm emissions]
3. How do plant-based and animal-based food products compare in terms of environmental impact? [multiple DVs]



#Problem Statement

The globalized "modern" large-scale food production system is doing great harm to the natural world, the atmosphere, and to the health and well-being of life on Earth. While the exalted lofty goals of neoliberal global food chains benefit the wealthiest populations of the globe, predominantly residing in the global North, by making many non-local foods available year-round, it is at the expense of transporation-related pollutants. Additionally, "modern" large-scale monoculture agricultural practices that involve machine-tilling and  emit vast amounts of carbon and GHG, which is helping fuel a climate change that affects global majority (non-white) people the hardest. By understanding which parts of the food production system are most harmful is an important steps in shaping policy to change those systems in ways that reduce their harm, as well as ultimately creates systems to support local, regenerative farming practices that help sequestor GHG, and thus help reverse climate change. This study expects that anmimal-based food products will be overall larger contributors to environmental harm, as well as seeks to examine which parts of the food production process are most harmful.

#Data Dimensions & Scales
##Variables
Food product (Kg CO2 emission / kg product)

Land use change (Kg CO2 emission / kg product)

Animal feed (Kg CO2 emission / kg product)

Farm (Kg CO2 emission / kg product)

Processing (Kg CO2 emission / kg product)

Transport (Kg CO2 emission / kg product)

Packaging (Kg CO2 emission / kg product)

Retail (Kg CO2 emission / kg product)

Total emissions (Kg CO2 emission / kg product)

Eutrophying emissions per 1000 kcal (gPO₄eq per 1000 kcal)

Eutrophying emissions per kilogram (gPO₄eq per kilogram)

Eutrophying emissions per 100g protein (gPO₄eq per 100 grams protein)

Freshwater withdrawals per 1000 kcal (liters per 1000 kcal)

Freshwater withdrawals per 100g protein (liters per 100g protein)

Freshwater withdrawals per kilogram (liters per kilogram)

Greenhouse gas emissions per 1000 kcal (kgCO₂eq per 1000 kcal)

Greenhouse gas emissions per 100g protein (kgCO₂eq per 100g protein)

Land use per 1000 kcal (m² per 1000 kcal)

Land use per kilogram (m² per kilogram)

Land use per 100g protein (m² per 100g protein)

Scarcity-weighted water use per kilogram (liters per kilogram)

Scarcity-weighted water use per 100g protein (liters per 100g protein)

Scarcity-weighted water use per 1000 kcal (liters per 1000 kilocalories)

##Related Definitions & Notes
Greenhouse gas emissions are measured in kilograms of carbon dioxide-equivalents. This means non-CO₂ gases are weighted by theamount of warming they cause over a 100-year timescale.

Scarcity-weighted water use represents freshwater use weighted by local water scarcity.This is measured in liters per kilogram of food product.

Eutrophication – the pollution of water bodies and ecosystems with excess nutrients – is a major environmental problem. The runoff of nitrogen and other nutrients from agricultural production systems is a leading contributor.

kcal = 1 calorie of macronutrient

##Summary of  the data

## Data Definition

**Environmental Impacts of Food Production** <br>
Data Collection: 2002 <br>
Source: ourworldindata.org

Contains the 43 most common foods grown across the globe and 23 columns as their respective land, water usage and carbon footprints.

**Source**<br>
Hannah Ritchie, Pablo Rosado, and Max Roser (2022) - “Environmental Impacts of Food Production” Published online at OurWorldinData.org. Retrieved from: 'https://ourworldindata.org/environmental-impacts-of-food

**Original Data Source**
Searchinger, T. D., Wirsenius, S., Beringer, T., & Dumas, P. (2018). Assessing the efficiency of changes in land use for mitigating climate change. Nature. – processed by Our World in Data

**Context** <br>
As the world’s population has expanded and gotten richer, the demand for food, energy and water has seen a rapid increase. Not only has demand for all three increased, but they are also strongly interlinked: food production requires water and energy; traditional energy production demands water resources; agriculture provides a potential energy source. This article focuses on the environmental impacts of food. Ensuring everyone in the world has access to a nutritious diet in a sustainable way is one of the greatest challenges we face.


In [ ]:
from google.colab import drive
drive.mount('/content/drive')

Mounted at /content/drive


In [ ]:
# Read the dataset
food_data = pd.read_excel('/content/drive/MyDrive/Data Science Bootcamp/Final Project/Food_Production_Impact_Full.xlsx')

# Create the Dataframe
FoodDataFrame = pd.DataFrame(food_data)

In [ ]:
# Import the libraries
import numpy as np                  # Scientific Computing
import pandas as pd                 # Data Analysis
import matplotlib.pyplot as plt     # Plotting
import seaborn as sns               # Statistical Data Visualization

# Let's make sure pandas returns all the rows and columns for the dataframe
pd.set_option('display.max_rows', None)
pd.set_option('display.max_columns', None)

# Force pandas to display full numbers instead of scientific notation
# pd.options.display.float_format = '{:.0f}'.format

# Library to suppress warnings
import warnings
warnings.filterwarnings('ignore')

In [ ]:
FoodDataFrame.head(7)

Food product  Land use change (KgCO2/Kg)  Animal Feed (KgCO2/Kg)  \
0  Wheat & Rye (Bread)                         0.1                     0.0   
1         Maize (Meal)                         0.3                     0.0   
2        Barley (Beer)                         0.0                     0.0   
3              Oatmeal                         0.0                     0.0   
4                 Rice                         0.0                     0.0   
5             Potatoes                         0.0                     0.0   
6              Cassava                         0.6                     0.0   

   Farm (KgCO2/Kg)  Processing (KgCO2/Kg)  Transport (KgCO2/Kg)  \
0              0.8                    0.2                   0.1   
1              0.5                    0.1                   0.1   
2              0.2                    0.1                   0.0   
3              1.4                    0.0                   0.1   
4              3.6                    0.1                   0.1   
5              0.2                    0.0                   0.1   
6              0.2                    0.0                   0.1   

   Packaging (KgCO2/Kg)  Retail (KgCO2/Kg)  Total_emissions (KgCO2/Kg)  \
0                   0.1                0.1                         1.4   
1                   0.1                0.0                         1.1   
2                   0.5                0.3                         1.1   
3                   0.1                0.0                         1.6   
4                   0.1                0.1                         4.0   
5                   0.0                0.0                         0.3   
6                   0.0                0.0                         0.9   

   Eutrophying emissions per 1000kcal (gPO₄eq per 1000kcal)  \
0                                                NaN          
1                                                NaN          
2                                                NaN          
3                                           4.281357          
4                                           9.514379          
5                                           4.754098          
6                                           0.708419          

   Eutrophying emissions per kilogram (gPO₄eq per kilogram)  \
0                                                NaN          
1                                                NaN          
2                                                NaN          
3                                              11.23          
4                                              35.07          
5                                               3.48          
6                                               0.69          

   Eutrophying emissions per 100g protein (gPO₄eq per 100 grams protein)  \
0                                                NaN                       
1                                                NaN                       
2                                                NaN                       
3                                           8.638462                       
4                                          49.394366                       
5                                          20.470588                       
6                                           7.666667                       

   Freshwater withdrawals per 1000kcal (liters per 1000kcal)  \
0                                                NaN           
1                                                NaN           
2                                                NaN           
3                                         183.911552           
4                                         609.983722           
5                                          80.737705           
6                                                NaN           

   Freshwater withdrawals per 100g protein (liters per 100g protein)  \
0                                                Na

In [ ]:
FoodDataFrame.tail(7)

Food product  Land use change (KgCO2/Kg)  Animal Feed (KgCO2/Kg)  \
36          Pig Meat                         1.5                     2.9   
37      Poultry Meat                         2.5                     1.8   
38              Milk                         0.5                     0.2   
39            Cheese                         4.5                     2.3   
40              Eggs                         0.7                     2.2   
41     Fish (farmed)                         0.5                     0.8   
42  Shrimps (farmed)                         0.2                     2.5   

    Farm (KgCO2/Kg)  Processing (KgCO2/Kg)  Transport (KgCO2/Kg)  \
36              1.7                    0.3                   0.3   
37              0.7                    0.4                   0.3   
38              1.5                    0.1                   0.1   
39             13.1                    0.7                   0.1   
40              1.3                    0.0                   0.1   
41              3.6                    0.0                   0.1   
42              8.4                    0.0                   0.2   

    Packaging (KgCO2/Kg)  Retail (KgCO2/Kg)  Total_emissions (KgCO2/Kg)  \
36                   0.3                0.2                         7.2   
37                   0.2                0.2                         6.1   
38                   0.1                0.3                         2.8   
39                   0.2                0.3                        21.2   
40                   0.2                0.0                         4.5   
41                   0.1                0.0                         5.1   
42                   0.3                0.2                        11.8   

    Eutrophying emissions per 1000kcal (gPO₄eq per 1000kcal)  \
36                                          31.958159          
37                                          26.324324          
38                                          17.750000          
39                                          25.418605          
40                                          15.111111          
41                                         131.351955          
42                                                NaN          

    Eutrophying emissions per kilogram (gPO₄eq per kilogram)  \
36                                              76.38          
37                                              48.70          
38                                              10.65          
39                                              98.37          
40                                              21.76          
41                                             235.12          
42                                                NaN          

    Eutrophying emissions per 100g protein (gPO₄eq per 100 grams protein)  \
36                                          47.206428                       
37                                          28.117783                       
38                                          32.272727                       
39                                          44.551630                       
40                                          19.610671                       
41                                         103.100197                       
42                                                NaN                       

    Freshwater withdrawals per 1000kcal (liters per 1000kcal)  \
36                                         751.380753           
37                                         356.756757           
38                                        1047.000000           
39                                        1448.372093           
40                                         401.180556           
41                                        2062.178771           
42                                                NaN           

    Freshwater withdrawals per 100g protein (liters per 100g protein)  \
36              

In [ ]:
FoodDataFrame.shape
rows=len(FoodDataFrame.axes[0])
columns=len(FoodDataFrame.axes[1])
print("Number of Rows: ", rows)
print("Number of Columns: ", columns)

Number of Rows:  43
Number of Columns:  23


In [ ]:
FoodDataFrame.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 43 entries, 0 to 42
Data columns (total 23 columns):
 #   Column                                                                   Non-Null Count  Dtype  
---  ------                                                                   --------------  -----  
 0   Food product                                                             43 non-null     object 
 1   Land use change (KgCO2/Kg)                                               43 non-null     float64
 2   Animal Feed (KgCO2/Kg)                                                   43 non-null     float64
 3   Farm (KgCO2/Kg)                                                          43 non-null     float64
 4   Processing (KgCO2/Kg)                                                    43 non-null     float64
 5   Transport (KgCO2/Kg)                                                     43 non-null     float64
 6   Packaging (KgCO2/Kg)                                                     43 

In [ ]:
#Calcuate total null values

initial_null_values=FoodDataFrame.isnull().sum().sum()
print(initial_null_values)

158


In [ ]:
#Calculate null values per column
FoodDataFrame.isnull().sum()

Food product                                                                0
Land use change (KgCO2/Kg)                                                  0
Animal Feed (KgCO2/Kg)                                                      0
Farm (KgCO2/Kg)                                                             0
Processing (KgCO2/Kg)                                                       0
Transport (KgCO2/Kg)                                                        0
Packaging (KgCO2/Kg)                                                        0
Retail (KgCO2/Kg)                                                           0
Total_emissions (KgCO2/Kg)                                                  0
Eutrophying emissions per 1000kcal (gPO₄eq per 1000kcal)                   10
Eutrophying emissions per kilogram (gPO₄eq per kilogram)                    5
Eutrophying emissions per 100g protein (gPO₄eq per 100 grams protein)      16
Freshwater withdrawals per 1000kcal (liters per 1000kcal)                  13
Freshwater withdrawals per 100g protein (liters per 100g protein)          17
Freshwater withdrawals per kilogram (liters per kilogram)                   5
Greenhouse gas emissions per 1000kcal (kgCO₂eq per 1000kcal)               10
Greenhouse gas emissions per 100g protein (kgCO₂eq per 100g protein)       16
Land use per 1000kcal (m² per 1000kcal)                                    10
Land use per kilogram (m² per kilogram)                                     5
Land use per 100g protein (m² per 100g protein)                            16
Scarcity-weighted water use per kilogram (liters per kilogram)              5
Scarcity-weighted water use per 100g protein (liters per 100g protein)     17
Scarcity-weighted water use per 1000kcal (liters per 1000 kilocalories)    13
dtype: int64

In [55]:
total_values=rows*columns
print(total_values)
percentage_null_values=(initial_null_values/total_values)*100


989


In [ ]:
#Return the percentage of null values with hundredths decimal place

print("The portion of null values is " , percentage_null_values.round(2) , "%")

The portion of null values is  15.98 %


In [ ]:
#Return the columns in this dataframe that have 2 (8.5%) or more missing values
#Note: Typically less than five percent missing values may not affect the results. More than 5% can be dropped, replaced with existing data, or imputed using mean or median.

missing_values = FoodDataFrame.isnull().sum()
columns_with_missing_values = missing_values[missing_values >= 2].index.tolist()
columns_with_missing_values


['Eutrophying emissions per 1000kcal (gPO₄eq per 1000kcal)',
 'Eutrophying emissions per kilogram (gPO₄eq per kilogram)',
 'Eutrophying emissions per 100g protein (gPO₄eq per 100 grams protein)',
 'Freshwater withdrawals per 1000kcal (liters per 1000kcal)',
 'Freshwater withdrawals per 100g protein (liters per 100g protein)',
 'Freshwater withdrawals per kilogram (liters per kilogram)',
 'Greenhouse gas emissions per 1000kcal (kgCO₂eq per 1000kcal)',
 'Greenhouse gas emissions per 100g protein (kgCO₂eq per 100g protein)',
 'Land use per 1000kcal (m² per 1000kcal)',
 'Land use per kilogram (m² per kilogram)',
 'Land use per 100g protein (m² per 100g protein)',
 'Scarcity-weighted water use per kilogram (liters per kilogram)',
 'Scarcity-weighted water use per 100g protein (liters per 100g protein)',
 'Scarcity-weighted water use per 1000kcal (liters per 1000 kilocalories)']

In [ ]:
number_variables_greater_than_85=len(columns_with_missing_values)
print(number_variables_greater_than_85)

14


In [46]:
#Create a new data frame for data adjustments
FoodDataFrame2 = pd.DataFrame(food_data)
FoodDataFrame2.head(7)

Food product  Land use change (KgCO2/Kg)  Animal Feed (KgCO2/Kg)  \
0  Wheat & Rye (Bread)                         0.1                     0.0   
1         Maize (Meal)                         0.3                     0.0   
2        Barley (Beer)                         0.0                     0.0   
3              Oatmeal                         0.0                     0.0   
4                 Rice                         0.0                     0.0   
5             Potatoes                         0.0                     0.0   
6              Cassava                         0.6                     0.0   

   Farm (KgCO2/Kg)  Processing (KgCO2/Kg)  Transport (KgCO2/Kg)  \
0              0.8                    0.2                   0.1   
1              0.5                    0.1                   0.1   
2              0.2                    0.1                   0.0   
3              1.4                    0.0                   0.1   
4              3.6                    0.1                   0.1   
5              0.2                    0.0                   0.1   
6              0.2                    0.0                   0.1   

   Packaging (KgCO2/Kg)  Retail (KgCO2/Kg)  Total_emissions (KgCO2/Kg)  \
0                   0.1                0.1                         1.4   
1                   0.1                0.0                         1.1   
2                   0.5                0.3                         1.1   
3                   0.1                0.0                         1.6   
4                   0.1                0.1                         4.0   
5                   0.0                0.0                         0.3   
6                   0.0                0.0                         0.9   

   Eutrophying emissions per 1000kcal (gPO₄eq per 1000kcal)  \
0                                                NaN          
1                                                NaN          
2                                                NaN          
3                                           4.281357          
4                                           9.514379          
5                                           4.754098          
6                                           0.708419          

   Eutrophying emissions per kilogram (gPO₄eq per kilogram)  \
0                                                NaN          
1                                                NaN          
2                                                NaN          
3                                              11.23          
4                                              35.07          
5                                               3.48          
6                                               0.69          

   Eutrophying emissions per 100g protein (gPO₄eq per 100 grams protein)  \
0                                                NaN                       
1                                                NaN                       
2                                                NaN                       
3                                           8.638462                       
4                                          49.394366                       
5                                          20.470588                       
6                                           7.666667                       

   Freshwater withdrawals per 1000kcal (liters per 1000kcal)  \
0                                                NaN           
1                                                NaN           
2                                                NaN           
3                                         183.911552           
4                                         609.983722           
5                                          80.737705           
6                                                NaN           

   Freshwater withdrawals per 100g protein (liters per 100g protein)  \
0                                                Na

#Understanding the data
There are 43 instances (rows), and 23 dimensions (columns). All of the variables are float 64 except for the name of the food product which is a string data type. There are a total of 158 null values out of a total of 989 possible total values, which equates to 15.98% missing data. Of the 23 variables, there are 14 that are missing >8.5% of data across all of the instances. All of the variables that capture carbon emissions (KgCO2/Kg) for parts of the food production process, as well as the aggregate carbon emissions (KgCO2/Kg) are complete (i.e. no missing values).

The other types of variables, which are eutrophying emissions across 3 types of scales, freshwater withdrawal across 3 types of scales, GHG emission across 2 types of scales, land use across 3 types of scales, and scarcity-weighted water use across 3 types of scales, are all missing significant amounts of values. The other scales represented within these variables measure aspects of environmental cost in terms of overall caloric value (per 1000kcal), protein value (per 100 grams protein), and overall product yield value (per kg of product.

One missing contextual element of this data is the difference between regenerative agricultural practices, which generally have a much smaller carbon footprint, and in fact often have a positive impact on the environment by seuestering carbon and nitrogen rather than releasing it, and practices that are destructive, such as monoculture, lack of crop rotation, use of chemical pesticides and fertilizer, and machine tilling. For the farming portion of the food production process, application of regenerative practices would lessen impact. The creators of the dataset acknowledge that there is large diversity of agricultural emissions, "in particular, beef, lamb, and dairy ...depending on how and where they’re produced." However, this is not nuance is not captured in the dataset. It would be very useful to collect data for both regenerative practices and for "conventional" practices to understand the difference , and thus, understand the value in investing in switching to regenertive practices.

##Feature Additions
Create a meta-variable for average farm-stage and post-farm emissions
Create Farm (3): Land-use change + Farming + Animal feed; Create
Post farm processing (4): Processing + Transport + Packaging + Retail. Also, remove all variables that are missing 2 or more values.




In [49]:
#Drop the 14 columns in FoodDataFrame2 that have more than 2 missing values

#Identify columns to drop
columns_to_drop = missing_values[missing_values > 2].index.tolist()
FoodDataFrame2 = FoodDataFrame2.drop(columns=columns_to_drop)

# Verify the changes
print(FoodDataFrame2.shape)
print(FoodDataFrame2.isnull().sum())

(43, 11)
Food product                      0
Land use change (KgCO2/Kg)        0
Animal Feed (KgCO2/Kg)            0
Farm (KgCO2/Kg)                   0
Processing (KgCO2/Kg)             0
Transport (KgCO2/Kg)              0
Packaging (KgCO2/Kg)              0
Retail (KgCO2/Kg)                 0
Total_emissions (KgCO2/Kg)        0
Farm Emissions (KgCO2/Kg)         0
Post Farm Emissions (KgCO2/Kg)    0
dtype: int64


In [54]:
#Add 2 columns for 2 aggregate variables: One for farm_emissions (3 variables) and one for post_farm_emissions (4 variables)

FoodDataFrame2['Farm Emissions (KgCO2/Kg)'] = (FoodDataFrame2['Land use change (KgCO2/Kg)'] + FoodDataFrame2['Animal Feed (KgCO2/Kg)'] + FoodDataFrame2['Farm (KgCO2/Kg)'])
FoodDataFrame2['Post Farm Emissions (KgCO2/Kg)'] = (FoodDataFrame2['Processing (KgCO2/Kg)'] + FoodDataFrame2['Transport (KgCO2/Kg)'] + FoodDataFrame2['Packaging (KgCO2/Kg)'] + FoodDataFrame2['Retail (KgCO2/Kg)'])

# Display the first few rows with the new columns
display(FoodDataFrame2.head())

Food product  Land use change (KgCO2/Kg)  Animal Feed (KgCO2/Kg)  \
0  Wheat & Rye (Bread)                         0.1                     0.0   
1         Maize (Meal)                         0.3                     0.0   
2        Barley (Beer)                         0.0                     0.0   
3              Oatmeal                         0.0                     0.0   
4                 Rice                         0.0                     0.0   

   Farm (KgCO2/Kg)  Processing (KgCO2/Kg)  Transport (KgCO2/Kg)  \
0              0.8                    0.2                   0.1   
1              0.5                    0.1                   0.1   
2              0.2                    0.1                   0.0   
3              1.4                    0.0                   0.1   
4              3.6                    0.1                   0.1   

   Packaging (KgCO2/Kg)  Retail (KgCO2/Kg)  Total_emissions (KgCO2/Kg)  \
0                   0.1                0.1                         1.4   
1                   0.1                0.0                         1.1   
2                   0.5                0.3                         1.1   
3                   0.1                0.0                         1.6   
4                   0.1                0.1                         4.0   

   Farm Emissions (KgCO2/Kg)  Post Farm Emissions (KgCO2/Kg)  
0                        0.9                             0.5  
1                        0.8                             0.3  
2                        0.2                             0.9  
3                        1.4                             0.2  
4                        3.6                             0.4

In [56]:
FoodDataFrame2.describe()

Land use change (KgCO2/Kg)  Animal Feed (KgCO2/Kg)  Farm (KgCO2/Kg)  \
count                   43.000000               43.000000        43.000000   
mean                     1.260465                0.453488         3.469767   
std                      3.357730                0.919200         7.083264   
min                     -2.100000                0.000000         0.100000   
25%                      0.000000                0.000000         0.350000   
50%                      0.200000                0.000000         0.800000   
75%                      0.800000                0.000000         2.200000   
max                     16.300000                2.900000        39.400000   

       Processing (KgCO2/Kg)  Transport (KgCO2/Kg)  Packaging (KgCO2/Kg)  \
count              43.000000             43.000000             43.000000   
mean                0.253488              0.195349              0.269767   
std                 0.372475              0.158795              0.342633   
min                 0.000000              0.000000              0.000000   
25%                 0.000000              0.100000              0.100000   
50%                 0.100000              0.100000              0.100000   
75%                 0.300000              0.200000              0.300000   
max                 1.300000              0.800000              1.600000   

       Retail (KgCO2/Kg)  Total_emissions (KgCO2/Kg)  \
count          43.000000                   43.000000   
mean            0.069767                    5.972093   
std             0.110270                   10.501753   
min             0.000000                    0.200000   
25%             0.000000                    0.850000   
50%             0.000000                    1.600000   
75%             0.150000                    6.000000   
max             0.300000                   59.600000   

       Farm Emissions (KgCO2/Kg)  Post Farm Emissions (KgCO2/Kg)  
count                  43.000000                       43.000000  
mean                    5.183721                        0.788372  
std                    10.099739                        0.692892  
min                     0.000000                        0.100000  
25%                     0.400000                        0.200000  
50%                     1.400000                        0.600000  
75%                     4.750000                        1.150000  
max                    57.600000                        2.400000

In [60]:
#Select only the numerical columns for correlation analysis
numerical_cols = FoodDataFrame2.select_dtypes(include=np.number).columns

#Calculate the correlation matrix
correlation_matrix = FoodDataFrame2[numerical_cols].corr()

#Display the correlation matrix
correlation_matrix

Land use change (KgCO2/Kg)  \
Land use change (KgCO2/Kg)                        1.000000   
Animal Feed (KgCO2/Kg)                            0.243624   
Farm (KgCO2/Kg)                                   0.646480   
Processing (KgCO2/Kg)                             0.466434   
Transport (KgCO2/Kg)                              0.052339   
Packaging (KgCO2/Kg)                              0.169467   
Retail (KgCO2/Kg)                                 0.182538   
Total_emissions (KgCO2/Kg)                        0.801875   
Farm Emissions (KgCO2/Kg)                         0.808027   
Post Farm Emissions (KgCO2/Kg)                    0.375585   

                                Animal Feed (KgCO2/Kg)  Farm (KgCO2/Kg)  \
Land use change (KgCO2/Kg)                    0.243624         0.646480   
Animal Feed (KgCO2/Kg)                        1.000000         0.572698   
Farm (KgCO2/Kg)                               0.572698         1.000000   
Processing (KgCO2/Kg)                         0.426774         0.704444   
Transport (KgCO2/Kg)                          0.231742         0.233778   
Packaging (KgCO2/Kg)                         -0.032542         0.156483   
Retail (KgCO2/Kg)                             0.514322         0.389291   
Total_emissions (KgCO2/Kg)                    0.574677         0.969024   
Farm Emissions (KgCO2/Kg)                     0.573658         0.968381   
Post Farm Emissions (KgCO2/Kg)                0.348289         0.571595   

                                Processing (KgCO2/Kg)  Transport (KgCO2/Kg)  \
Land use change (KgCO2/Kg)                   0.466434              0.052339   
Animal Feed (KgCO2/Kg)                       0.426774              0.231742   
Farm (KgCO2/Kg)                              0.704444              0.233778   
Processing (KgCO2/Kg)                        1.000000              0.338420   
Transport (KgCO2/Kg)                         0.338420              1.000000   
Packaging (KgCO2/Kg)                         0.397291              0.084875   
Retail (KgCO2/Kg)                            0.428702             -0.008222   
Total_emissions (KgCO2/Kg)                   0.719673              0.224505   
Farm Emissions (KgCO2/Kg)                    0.687960              0.202448   
Post Farm Emissions (KgCO2/Kg)               0.879808              0.451762   

                                Packaging (KgCO2/Kg)  Retail (KgCO2/Kg)  \
Land use change (KgCO2/Kg)                  0.169467           0.182538   
Animal Feed (KgCO2/Kg)                     -0.032542           0.514322   
Farm (KgCO2/Kg)                             0.156483           0.389291   
Processing (KgCO2/Kg)                       0.397291           0.428702   
Transport (KgCO2/Kg)                        0.084875          -0.008222   
Packaging (KgCO2/Kg)                        1.000000           0.013043   
Retail (KgCO2/Kg)                           0.013043           1.000000   
Total_emissions (KgCO2/Kg)                  0.205018           0.391958   
Farm Emissions (KgCO2/Kg)                   0.163125           0.380518   
Post Farm Emissions (KgCO2/Kg)              0.729594           0.394165   

                                Total_emissions (KgCO2/Kg)  \
Land use change (KgCO2/Kg)                        0.801875   
Animal Feed (KgCO2/Kg)                            0.574677   
Farm (KgCO2/Kg)                                   0.969024   
Processing (KgCO2/Kg)                             0.719673   
Transport (KgCO2/Kg)                              0.224505   
Packaging (KgCO2/Kg)                              0.205018   
Retail (KgCO2/Kg)                                 0.391958   
Total_emissions (KgCO2/Kg)                        1.000000   
Farm Emissions (KgCO2/Kg)                         0.998499   
Post Farm Emissions (KgCO2/Kg)                    0.602082   

                                Farm Emissions (KgCO2/Kg)  \
Land use change (KgCO2/Kg)                       0.808027   
Animal Feed (KgCO2/Kg)

##Resources and References

##Peer-Reviewed Journal Research
Crippa, M., Solazzo, E., Guizzardi, D., Monforti-Ferrario, F., Tubiello, F. N., & Leip, A. J. N. F. (2021). Food systems are responsible for a third of global anthropogenic GHG emissions. Nature Food, 2(3), 198-209

Clark, Michael A., Nina GG Domingo, Kimberly Colgan, Sumil K. Thakrar, David Tilman, John Lynch, Inês L. Azevedo, and Jason D. Hill. “Global food system emissions could preclude achieving the 1.5° and 2° C climate change targets.” Science, 370, no. 6517 (2020): 705-708.

## Non-Profit Organizations and Non-Governmental Organizations
Report: "Food production is responsible for one-quarter of the world’s greenhouse gas emissions"
https://ourworldindata.org/food-ghg-emissions
Excerpt: "Crop production accounts for 27% of food emissions...21% of food’s emissions comes from crop production for direct human consumption, and 6% comes from the production of animal feed. They are the direct emissions which result from agricultural production – this includes elements such as the release of **nitrous oxide from the application of fertilizers and manure; methane emissions from rice production; and carbon dioxide from agricultural machinery.**"

Report: Cultivating Climate Resilience on Farms and Ranches. Sustainable Agriculture Research and Education (SARE).

## Federal, State, Local Government


##Book Chapters
*Resilient Agriculture: Cultivating Food Systems for a Changing Climate* by Laura Lengnick.

Introduction. *Farming While Black: Soul Fire Farm's Practical Guide to Liberation on the Land* by Leah Penniman (2018).

Chapter 7: Tools & Technology. *Farming While Black: Soul Fire Farm's Practical Guide to Liberation on the Land* by Leah Penniman (2018).